rpc/trace.go

```diff
@@ -141,7 +142,7 @@ func (h *Handler) traceTransaction(ctx context.Context, hash *felt.Felt) (*vm.Tr
 	httpHeader := http.Header{}
 	httpHeader.Set(ExecutionStepsHeader, "0")
 
-	if err != nil {
+	if err != nil && !errors.Is(err, db.ErrKeyNotFound) {
```

Comment:
super nit

can't it be written as:

```suggestion
	if !errors.Is(err, db.ErrKeyNotFound) {
```

Reply:
I guess it's common go pattern we use it everywhere in Juno (see [node.go](https://github.com/NethermindEth/juno/blob/d3ff578e4bbe21c85c7718986490107ab40a2a1d/node/node.go#L137), [trie.go](https://github.com/NethermindEth/juno/blob/d3ff578e4bbe21c85c7718986490107ab40a2a1d/core/trie/trie.go#L67), [sync.go](https://github.com/NethermindEth/juno/blob/d3ff578e4bbe21c85c7718986490107ab40a2a1d/p2p/sync/sync.go#L339) etc.)

Reply:
I think if you remove the first check then a `nil` err will turn true the second condition, which might not be the intended behaviour